Adafruit_SSD1306.cpp

```diff
@@ -192,10 +192,17 @@ void Adafruit_SSD1306::begin(uint8_t vccstate, uint8_t i2caddr, bool reset) {
   {
     // I2C Init
     Wire.begin();
-#ifdef __SAM3X8E__
+#if defined(__SAM3X8E__)
     // Force 400 KHz I2C, rawr! (Uses pins 20, 21 for SDA, SCL)
     TWI1->TWI_CWGR = 0;
     TWI1->TWI_CWGR = ((VARIANT_MCK / (2 * 400000)) - 4) * 0x101;
+#elif defined(__SAMD21G18A__)
+    // Last I checked, Wire.setClock is a dummy function on the Arduino Zero.
+    // This seems to work instead, but maybe there's a better way?
+    // Configure 400khz I2C
+    sercom3.disableWIRE();  // BAUD is an ENABLE-protected register.  Disable it frst
+    SERCOM3->I2CM.BAUD.bit.BAUD = (VARIANT_MCK / ( 2 * 400000) - 1);
+    sercom3.enableWIRE();   //Re-enable I2C
```

Comment:
I just checked, and `Wire.setClock` isn't a dummy function, at least in Arduino 1.6.7 with version 1.6.2 of the SAMD board kit.